main/badge/issued.php

```diff
@@ -59,6 +59,15 @@
     'courses' => []
 ];
 
+// Open Graph Markup
+$htmlHeadXtra[] = "
+    <meta property='og:type' content='article' />
+    <meta property='og:title' content='".sprintf(get_lang('IHaveObtainedSkillXOnY'), $skillInfo['name'], api_get_setting('siteName'))."' />
+    <meta property='og:url' content='".api_get_path(WEB_PATH)."badge/".$skillId."/user/".$userId."' />
+    <meta property='og:description' content='".$skillInfo['description']."' />
+    <meta property='og:image' content='".$skillInfo['badge_image']."' />
+";
+
```

Comment:
Fixed now , i validate the twitter:card to replace the description, title and url to the actual badge page if is in this page and if twitter:card is enable